presto-main/src/main/java/com/facebook/presto/memory/MemoryResource.java

```diff
@@ -48,4 +50,12 @@ public MemoryInfo getMemoryInfo(MemoryPoolAssignmentsRequest request)
         taskManager.updateMemoryPoolAssignments(request);
         return memoryManager.getInfo();
     }
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    public MemoryInfo getMemoryInfo()
+    {
+        taskManager.updateMemoryPoolAssignments(new MemoryPoolAssignmentsRequest("", 0, new ArrayList<>()));
```

Comment:
I will add TestMemoryResource.java with test cases for current POST call and this GET call. Do you want to change/modify the requirements for GET call though (What should the empty request mean?)